packages/ocapn/src/client/types.js

```diff
@@ -0,0 +1,98 @@
+// @ts-check
```

Comment:
Elsewhere, we’re moving in the direction of writing all-types files as `.ts`. This is not a hard line but one we can move incrementally.